### DISCUSSION TITLE
Way to specify band in python version?

### DISCUSSION THREAD

**Discussion by tm-jc-nacpil** [isciences/exactextract#109] [Q&A] [answered]
Hi @dbaston ! I'd like to ask if there's currently a way to specify which band to aggregate when using the python functions? I see there's a way when using the CLI, but would be great if we can specify the band in Python as well

also thanks for the great work 😄 we've been trying out this package and we see a lot of upsides vs other zonal statistic packages 🙌

**Comment by dbaston** [comment, marked as answer]
You can create a `RasterSource` and pass that to `exact_extract` instead of the filename. For example, if you are using rasterio, you could do

```
from exactextract.raster import RasterioRasterSource
exact_extract(RasterioRasterSource(rast, 2), poly, 'sum')
```

**Reply by tm-jc-nacpil** [reply]
This works great, thanks! :D
